tests/foreman/api/test_docker.py

```diff
@@ -679,11 +699,24 @@ def test_publish_multiple_docker_repo_content_view(self):
 
         @Feature: Docker
 
-        @Status: Manual
-
         """
+        product = entities.Product(organization=self.org).create()
+        repo = _create_repository(product)
+        content_view = entities.ContentView(
+            composite=False,
+            organization=self.org,
+        ).create()
+        content_view.repository = [repo]
+        content_view = content_view.update(['repository'])
+        self.assertIn(repo.id, [repo_.id for repo_ in content_view.repository])
```

Comment:
`content_view` only has one repository, right? Consequently, this would be a more precise and concise assertion:

```python
self.assertEqual([repo.id], [content_view.repository[0].id])
```